universal-application-tool-0.0.1/conf/messages

```diff
@@ -72,6 +72,12 @@ link.externalLink=External site
 # The title of the page for the list of programs.
 title.programs=Programs & services
 
+# Subtitle for the list of programs with draft applications
+titles.inProgressPrograms=Continue
+
+# Subtitle for the list of active programs
```

Comment:
nit: "active programs" is kind of confusing because in the context of applications, "active" makes me think of "in progress. Suggest "Subtitle for the list of programs for which the applicant has no draft applications."